### DISCUSSION TITLE
Unlocking accelerated innovation without compromising customer trust 🚀  [SEC2846M]

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#74968] [Code Security]
**Speakers:** 
- Vincent King (Head of Secure Cloud Transformation, Bank of England)
- @mph4 (Mike Hanley, Chief Security Officer and SVP of Engineering, GitHub)
- Sigurd Pollmann (Head of DevSecOps Toolchain & Operations, Deutsche Börse Group)
- Helen Lau (Head of Engineering Tooling, Commonwealth Bank of Australia)
- Gerard McMahon (Head of ALM Tools and Platforms, Fidelity Investments)

**Time:** Thursday, Nov 9, 1:45 PM - 2:25 PM PST

> Software development is a dynamic industry, and achieving accelerated growth relies on elevating the developer experience to unlock innovation, while steadfastly upholding customer trust. These seemingly opposing concepts beg the question: Can you truly attain business agility, developer contentment, enhanced productivity, seamless collaboration, and unwavering security all at once? Join us in this session for insights from GitHub's Chief Security Officer and SVP of Engineering, Michael Hanley, and experts in heavily regulated sectors—like banking and finance—from Fidelity, CBA, Deutsche Börse Group, and Bank of England as they share their proven strategies for effectively building, scaling, and fortifying software in today's swiftly evolving digital landscape.

Thank you for joining the session! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do our best to answer.

To watch the recording, please click [`link tbd`](https://github.com/orgs/community/discussions/74968).👈

**Comment by MOHAMoPOCO** [comment]
[dumpstate-stats.txt](https://github.com/community/community/files/13787788/dumpstate-stats.txt)
